www/src/translation.ts

```diff
@@ -195,6 +195,11 @@ const dictionary: {[key: string]: DictionaryEntry} = {
         en: "Primed",
         ja: "長持続"
     },
+    GOOD_OMEN: {
+        en: "Good Omen",
+        ja: "良兆候",
+        ko: "양징후",  // todo: fix this once 6.4 is released in Korean servers.
```

Comment:
```suggestion
        ko: "좋은 징후",  // todo: fix this once 6.4 is released in Korean servers.
```
It might become something other words, but I feel this is better than '양징후'